tasks/sips/args/SIPArgs.ts

```diff
@@ -167,11 +167,37 @@ const sip0072 = async (
   return args;
 };
 
+const SIPSOV3564 = async (hre): Promise<ISipArgument> => {
+  const { ethers } = hre;
+  const mocIntegrationProxy = await ethers.getContract("MocIntegration"); // MocIntegration
+  const newMocIntegrationImpl = await ethers.getContract("MocIntegration_Implementation");
+  const myntAdminProxy = await ethers.getContract("MyntAdminProxy");
+
```

Comment:
pls add validation of deployments - they should differ from the current implementation used by proxy